### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by milen-ale**
Add the desktop version of the personal portfolio

   - Added responsive units to the mobile version
   - Added a media query for website responsiveness at 768px;
   -  Added top menu link for easy navigation to different sections of the webpage
   - Realigned project cards with CSS grid in the desktop version to allow for reordering
    Styled the elements accordingly

### CODE REVIEW

File: style/desktop-style.css [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  .card-title-two {
+    display: flex;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 2rem;
+    line-height: 5rem;
+    color: #091e42;
+    padding: top 0;
+    margin: 0;
+  }
```
Comment (tresorsawasawa) [inline style/desktop-style.css]: - [ ] Kindly make sure you improve your **projects cards** styling to make them stick to the Figma design. I noticed that from Figma template elements in each block in a card used half of the `width` i.e. **Image block width 50%** and **text block 50%**. I suggest making the same on your project
- [ ] Also, make sure to use the correct `spacing` around the `cards` (**spacing at the right and left of your cards**) as from the Figma design. And your cards will stick to the template design.
#### Figma design
card_fig

#### Your design
card_st

File: style/desktop-style.css [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    align-items: center;
+    width: 500px;
+  }
+
+  .email-field,
+  .text-field,
+  .textarea-field {
+    width: 500px;
+  }
+}
```
Comment (tresorsawasawa) [inline style/desktop-style.css]: - [ ] Kindly make sure you used the correct properties for each form's element to make your project stick to the Figma design, like `paddings` for text in `inputs` and `textarea` fields, correct `color` for **text**, and `inputs && text area` border-radius.
#### Figma Design
form_padd
#### Your Design
form_padding

File: style/desktop-style.css [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+  .about-me1 {
+    width: 50%;
+    margin-right: 46px;
+  }
+
+  .language-fream ul {
+    display: flex;
+    flex-direction: row;
+  }
```
Comment (tresorsawasawa) [inline style/desktop-style.css]: - [ ] KIndly make sure you used the correct `width` for each block in the **About me section**. From the Figma design, there are 2 blocks, the left one takes 40% and the right one the remaining 60% of the page's width, and the contents of each of them are aligned to the right. Try to fix this and your project will stick to the Figma design.
- [ ] Kindly make sure you use the correct **color** on titles text like **Languages**(and its content), **Skills**, and **Frameworks** to make your project stick to the Figma design.
### Figma design
ab_fi
### Your design
ab_st

File: index.html [hunk 4]
```diff
@@ -7,11 +7,17 @@
...
-      <a href="#" class="my-logo"> Milen </a>
+      <a href="#" class="my-logo"> <h3>Milen</h3> </a>
+      <ul>
+        <li><a href="#">Portfolio </a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#"> Contact</a></li>
+      </ul>
       <nav>
         <div class="my-button">
           <img alt="nav" class="menu" src="./images/Enabled.png" />
```
Comment (tresorsawasawa) [inline index.html:23]: - [ ] Make sure to improve the header styling to make it stick to the Figma design. I think you can check on the **spacing** at right and left, **color** and **spacing** of the **nav links** (portfolio, about, contact,...), and the **Fonts**.
#### Figma design
st_f
#### Your design
st_h

File: index.html [hunk 5]
```diff
@@ -20,19 +26,13 @@
...
-          and<br />
-          experience! If you like what you see and<br />
-          have a project you need coded, don’t<br />
-          hestiate to contact me.
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
         </p>
       </div>
```
Comment (tresorsawasawa) [inline index.html]: - [ ] Kindly make sure the **hading section** design sticks to the one from the Figma template like on elements positon, `height`, `spacing` between and around elements(top, right, bottom and left), `fonts`,...
### Figma design
head_sect
### Your design
head_sec

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @milen-ale 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
Hi @milen-ale , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**